### ISSUE TITLE
Add an option to remove the back button from Start Menu.

### ISSUE THREAD

**Issue by EternallyAries** [Open-Shell/Open-Shell-Menu#414]
When you go into "Main Menu" tab and turn on the option under "All Program style" called "Open Automatically" would it be possible to add the option to also remove the back button from the start menu?

**Comment by Ibuprophen** [comment]
Hmm... I think I know what your question is, but can you please elaborate a touch (maybe an image example too) so there's no assumptions to your question? 


~Ibuprophen

**Comment by bonzibudd** [comment]
In the "All Programs" view? There are a few ways to do that by modifying the skin file, but it will also remove the "All Programs" button in the main view (On a Windows 7 style skin). Let us know which skin you are using.

**Comment by EternallyAries** [comment]
Hello, so yeah I'm talking about the "All Programs" button in general. I can most definitely edit the skin file if needed, but I believe an option could be a welcome addition though. Even if most users wouldn't have a need for it. As for the button in question Ibuprophen, I'm talking about this: 
![download](https://user-images.githubusercontent.com/36057788/89620240-3d893780-d844-11ea-9eb6-57c3f1aa347f.png)

I don't mind having the "All Programs" button removed entirely to have the "Back" button be removed since I'd wouldn't have a need to use it at all. If this can be added as an option, that would be awesome, if not I'd be happy to have an idea how to edit the style to remove it.

Thank you.

**Comment by ge0rdi** [comment]
@EternallyAries 
I think it is now clear what you meant by your enhancement request (screenshots always help a lot).

I'm just wondering what's the problem with `Back` button.
Does it somehow interfere with your workflow?
Or is it just that you don't like the button there (cosmetic issue)?

**Comment by EternallyAries** [comment]
Yeah I have no issues with my workflow at all, and honestly it isn't a problem. I guess it's more of a I don't like having it there if I don't have any use for it. Granted I am most likely a very small minority of users who would like to have an option to out right remove the option from the Startmenu as a whole if there no reason to ever use it.

The best way to put it, it's far from in my way and it doesn't interfere in any way what so ever, buuuut due to me love having as many options as possible, having the ability just to hide/remove/disable the button as a whole would make it look a whole lot better in my book since I have no need to use the other menu since I never pin any of my stuff in that menu haha.

Example below: 
![Screenshot_1](https://user-images.githubusercontent.com/36057788/89629141-30734500-d852-11ea-955d-891b1584990d.png)

I think it should be possible to do something like this

Proof of concept example below:
![Screenshot_2](https://user-images.githubusercontent.com/36057788/89629289-6dd7d280-d852-11ea-9111-25516faa5628.png)

I tried looking into the skin files to remove it, but since I have no experience with skins at the moment, I can't seem to find a way to out right remove the button.

So a simple setting like this placed here would work wonders for people who would like to have the option.

Proof of concept example below 2:
![Screenshot_3](https://user-images.githubusercontent.com/36057788/89630058-a4fab380-d853-11ea-9d57-450d8a82b97a.png)

I use a paint program to add it in as a proof of concept. I think it could work and after all, Open-Shell is design for customization purposes.

I appreciate your folks time btw, and I think it would be really cool to have this as an option for Open-Shell down the road for people like me who likes to have things in our startmenu to look as clean and tidy as possible haha.

**Comment by CTVCAM8** [comment]
You might be the one in a million who wants this feature. :)
It could be done somewhat modifying the skin file.

In .skin7 file there are 4 items that need to be made invisible:

The All Programs icon-easy-make bitmap 7 invisible 

All Programs button-easy-make a separate invisible button for that location

Main separator-easy-make invisible. 
The problem with that is the main separator will be invisible at any location on the first column.

"All Programs" and "Back" text-Make not visible-use extreme text padding to push the text out of way.
I tried it. Seems to work. Can still click on the area to go back (an area close to bottom separator).
I tried to make everything small to be a small invisible area. No difference-still big.
I put as an option in this modified Metro.skin7 Have to unzip it then put in the skins folder.
Then pick "Metro Invisible All Programs  area" in the skin dropdown list in the skin settings tab.


[Metro Invisible All Programs  area .zip](https://github.com/Open-Shell/Open-Shell-Menu/files/5043032/Metro.Invisible.All.Programs.area.zip)



If you want to open up the skin file, I put the added text at the bottom of the file.
Note. I did not test if it hides the new installed programs colored button.

**Comment by EternallyAries** [comment]
Ayyy @CTVCAM8 this works wonderfully even though the button is still kinda there. This is definitely what I'm looking for though. I'll probably dabble with the skin file you made to see what else can be done and how you did it. But otherwise this is a great start. If anyone else got any ideas how to completely disable the button that would be wonderful.

Thank you so much though CTV.

**Comment by bonzibudd** [comment]
I could actually see where the appeal for this is, kind of a similar setup to how the Windows 10 menu functions in new versions, with the programs view instantly visible. The next step for a setup like this would be for an option to combine the recent programs and all programs view.

That is probably a topic for another day, however.

**Comment by EdePC** [comment]
Disable All Programs button, Nope!
Disable Back Button and set All Programs by default, Yes!, but no loose Searchbar focus, please XD

**Comment by LZeugirdor** [comment]
Would definitely love this as a feature. +1 for me. Miss the way the old windows 10 start menu was and attempted to replicate it, got pretty close. 

![Image](https://github.com/user-attachments/assets/bccdc62d-fba9-4532-abed-79516ae50f58)

**Comment by ConsoleScript** [comment]
This is a major issue that arose when we added it for 200 employees. Nearly half complained about it the next day because moving the mouse over it without clicking causes all apps to close. This is an extremely major bug. There has to be a button removal option IMHO.

**Comment by CTVCAM8** [comment]
Would keeping the "back" button and just slowing down the mouse hover speed help in some way?


<img width="957" height="590" alt="Image" src="https://github.com/user-attachments/assets/1182085c-2120-4cc0-ab13-89023b898de7" />

**Comment by LZeugirdor** [comment]
> Would keeping the "back" button and just slowing down the mouse hover speed help in some way?
> 
> 
> <img width="957" height="590" alt="Image" src="https://github.com/user-attachments/assets/1182085c-2120-4cc0-ab13-89023b898de7" />
> 

Don't use windows anymore, got tired of the BS. But I think I solved this by setting some delays to -1. Give that a shot and see if hover activate is disabled.
